safe/report/test/test_impact_report.py

```diff
@@ -980,8 +980,10 @@ def test_qgis_map_pdf_report(self):
         # insert layer to registry
         layer_registry = QgsMapLayerRegistry.instance()
         layer_registry.removeAllMapLayers()
-        rendered_layer = impact_function.exposure_impacted
+        rendered_layer = impact_function.impact
         layer_registry.addMapLayer(rendered_layer)
+        for layer in impact_function.outputs:
+            layer_registry.addMapLayer(layer)
```

Comment:
Ah yes, I think I overlook this. I will check different parts too.
Thanks for the heads up